### PR TITLE
Add a z-index to js-plotly-tester as a workaround for a chrome bug 

### PR DESCRIPTION
**Pull request by benjeffery**
Proposed solution for https://github.com/plotly/plotly.js/issues/513.

### CODE REVIEW

File: src/components/drawing/index.js [hunk 1]
```diff
@@ -462,7 +462,8 @@ drawing.makeTester = function(gd) {
             left: '-10000px',
             top: '-10000px',
             width: '9000px',
-            height: '9000px'
+            height: '9000px',
+            'z-index': '1'
```
Comment (etpinard) [inline src/components/drawing/index.js:466]: Really? That was it? Nice fix.

**Comment by etpinard** [comment]
@benjeffery great. 

I'm trying to decide whether we should add a test for this very specific case. I'm leaning toward to _no_ side. In any case, PR away!

File: src/components/drawing/index.js (hunk 1 above)
Comment (benjeffery) [inline src/components/drawing/index.js:466]: I have no understanding of _why_ it works. But thankfully it does.

**Comment by benjeffery** [comment]
PR done upstream - closing.

**Pull request closed without merging**